easybuild/framework/easyblock.py

```diff
@@ -1423,6 +1423,22 @@ def make_module_extra(self, altroot=None, altversion=None):
                                      value, type(value))
             lines.append(self.module_generator.append_paths(key, value, allow_abs=self.cfg['allow_append_abs_path']))
 
+        # Add automatic PYTHONPATH or EBPYTHONPREFIXES if they aren't already present
+        if 'PYTHONPATH' not in self.module_generator.added_paths_per_key and \
+           'EBPYTHONPREFIXES' not in self.module_generator.added_paths_per_key:
+            python_paths = [path for path in glob.glob('lib*/python*/site-packages')
+                            if re.match(r'lib(64)?/python\d+\.\d+/site-packages', path)]
+            use_ebpythonprefixes = get_software_root('Python') and build_option('prefer_ebpythonprefixes') and \
+                self.cfg['prefer_ebpythonprefixes']
+
+            if len(python_paths) > 1 and not use_ebpythonprefixes:
+                raise EasyBuildError('Multiple python paths requires EBPYHONPREFIXES: ' + ', '.join(python_paths))
+            elif python_paths:
+                if use_ebpythonprefixes:
+                    lines.append(self.module_generator.append_paths('EBPYHONPREFIXES', '.'))
+                else:
+                    lines.append(self.module_generator.append_paths('PYTHONPATH', python_paths))
```

Comment:
@Micket Should be `prepend_paths`, not `append_paths`?

Reply:
indeed it should